lib/IDE/Refactoring.cpp

```diff
@@ -1213,10 +1213,32 @@ bool RefactoringActionExtractFunction::performChange() {
 
     OS << " {\n";
 
+    SourceLoc RangeStart = RangeInfo.ContentRange.getStart();
+    auto ContentIndentation = SM.getLineAndColumn(RangeStart).second;
```

Comment:
Rather than storing the column and adding spaces, could you store and use the indentation of the line containing RangeStart instead? There's an existing API in Lexer.h you can use: 
```
  /// Retrieve the string used to indent the line that contains the given
  /// source location.
  ///
  /// If \c ExtraIndentation is not null, it will be set to an appropriate
  /// additional intendation for adding code in a smaller scope "within" \c Loc.
  static StringRef getIndentationForLine(SourceManager &SM, SourceLoc Loc,
                                         StringRef *ExtraIndentation = nullptr);
```